docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -183,7 +174,7 @@ Modules & Plugins
 Other directories
 -----------------
 
-Collections MUST not use files outside ``meta/``, ``plugins/``, ``roles/`` and ``playbooks/`` in any plugin, role, or playbook that can be called by FQCN, used from other collections, or used from user playbooks and roles. A collection must work if every file or directory is deleted from the installed collection except those four directories and their contents.
```

Comment:
Please re-add this sentence. It is extremely vital to not destroy collections when building the Ansible package.

Reply:
done. restructured it a bit, thanks